### DISCUSSION TITLE
how to remove the sector in SubmitPreCommitBatch status

### DISCUSSION THREAD

**Discussion by olinkcloud** [filecoin-project/lotus#9392] [General Q&A] [answered]
Hello,
I just run a lotus-miner sectors pledge to test the pc1 and pc2 progress and performance.
Once everything is done, the sectors is in SubmitPreCommitBatch status.
Now I want to remove these sectors and start over, how can I do that? 
I used lotus-miner sectors remove --really-do-it
But it doesn't work.

**Comment by rjan90** [comment]
Hey @olinkcloud! When the sectors are in SubmitPreCommitBatch, they PreCommitBatch is waiting to be sent to the chain. You can manually publish this message with: `lotus-miner sectors batching precommit --publish-now`, after that message is sent the sectors will move onto the WaitSeed stage and be there for approximately 90 minutes, before the C2-tasks start.

Since you already have remove the sectors with `lotus-miner sectors remove --really-do-it` those should take effect after the message has been sent. In general its not really adviced to remove a sector during sealing, as its mostly used for troubleshooting if something goes wrong.¨

**Comment by rjan90** [comment, marked as answer]
By the way, if you are only planning to test the performance of a single sealing task, I recommend you to check out the single sealing task benchmarking tool https://lotus.filecoin.io/storage-providers/operate/benchmarks/#single-task-benchmark
